package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@okta/okta-signin-widget",
   "description": "The Okta Sign-In Widget",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
This seems incorrect. This version should be `3.0.1` Similar to https://github.com/okta/okta-signin-widget/pull/721/commits.

Reply:
Sorry this is not correct PR